.vscode/settings.json

```diff
@@ -17,9 +17,7 @@
         "colorpicker",
         "CTSL",
         "cupertino",
-        "Dalmax",

```

Comment:
The items I'm removing here appear to be stale. But check-spelling and cspell were effectively pinning each other's stale items preventing either from being able to garbage collect.